### PR TITLE
Astronomer Software Release Notes - 2022/08/05

### LINKED ISSUE #1003: Astronomer Software Release Notes - 2022/08/05
Create Astronomer Software Release Notes for 0.29.3

### PR DESCRIPTION
**Pull request by kwcanuck**
Resolves #1003

### CODE REVIEW

File: software/release-notes.md [hunk 1]
```diff
@@ -14,6 +14,18 @@ This document includes all release notes for Astronomer Software version 0.29.
 
 We're committed to testing all Astronomer Software versions for scale, reliability and security on Amazon EKS, Google GKE and Azure AKS. If you have any questions or an issue to report, don't hesitate to [reach out to us](https://support.astronomer.io).
 
+## v0.29.3
+
+Release date: August 8, 2022
```
Comment (paolaperaza) [inline software/release-notes.md:19]:
```suggestion
Release date: August 5, 2022
```

File: software/release-notes.md [hunk 2]
```diff
@@ -14,6 +14,18 @@ This document includes all release notes for Astronomer Software version 0.29.
...
+
+### Additional improvements
+
+- Resolved several high and critical CVEs.
+
+### Bug fixes
+
+- GraphQL extra AU queries now return results for all Deployments on Airflow.
```
Comment (paolaperaza) [inline software/release-notes.md:27]:
```suggestion
- API requests to query the AUs allocated to **Extra Capacity** are now returning results for all Deployments in a Workspace. Previously, queries were only returning partial results.
```

**Review by paolaperaza — APPROVED** [review]
@vishwas-astro Could you review?

I left two suggestions based on @kwcanuck's changes -- if they look good to you, I'll commit them and merge

**Review by vishwas-astro — APPROVED** [review]

File: software/release-notes.md (hunk 2 above)
Comment (vishwas-astro) [reply software/release-notes.md:27]: Agree to Paola's suggestion.

**Pull request merged**